whatis277/harvest
# bean: Fix failing integration tests

Integration tests were failing because

`Query` doesn't return `pgx.ErrNoRows`
So we need to check for it manually

We forgot to update the subs tests
That happened when modifying the subs seed data

Testing instructions:
1. `dc down --volumes`
2. `dc up bean`
3. `dc exec -e INTEGRATION=1 bean go test ./... -v`
4. Ensure all data source tests aren't skipped
5. Ensure all data source tests pass
